packages/server-core/src/createCorsServer.ts

```diff
@@ -0,0 +1,42 @@
+import cors_proxy from 'cors-anywhere'
+import config from './appconfig'
+import fs from 'fs'
+import net from 'net'
+
+const createCorsServer = (port: number) => {
```

Comment:
Our use of SSL is different between development and production, as can be seen in server/src/index.ts; this will need to be accounted for here. In dev mode, we have the server use SSL, so that we can simulate a production environment with https://URLs. When in dev mode, the https server uses the self-signed certs that are pointed to here.

In production, we start up the servers with SSL off, and do SSL termination at the load balancer. Unless there's something I'm missing, the cors-anywhere server should not have httpOptions if the server's being started in production mode - again, see server/src/index.ts for determining whether SSL should be turned on.